pkg/master/master.go

```diff
@@ -438,6 +441,9 @@ func (m *Master) initV1ResourcesStorage(c *Config) {
 
 		"componentStatuses": componentstatus.NewStorage(func() map[string]apiserver.Server { return m.getServersToValidate(c) }),
 	}
+	if !m.ApiGroupVersionOverrides["autoscaling/v1"].Disable {
```

Comment:
Same comment from other PR.

If we *want* these to be turned on and off with HPA, then I retract the comment. @bgrant0607 do we want scale subresource to only appear when HPA is turned on?